src/index.html

```diff
@@ -21,7 +21,345 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="Логотип"
+        class="logo_img"
+      />
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="./src"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="./src"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="./src"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="./src"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="./src"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="./src"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="./src"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="./src"
```

Comment:
All navigation links have the same `href` attribute (`'./src'`). Please update these links to point to the correct destination pages to ensure proper navigation functionality.